core/tools/generate_names.cc

```diff
@@ -361,6 +361,14 @@ NameDef names[] = {
     {"selfNew", "<self-new>"},
     {"stringInterpolate", "<string-interpolate>"},
 
+    // Packager
+    {"import_", "import"},
+    {"export_", "export"},
```

Comment:
Are these actually C++ reserved keywords?

Reply:
Looks like `export` is, `import` isn't unless it's a preprocessor directive.

https://en.cppreference.com/w/cpp/keyword

I can see if I can change `import`